### DISCUSSION TITLE
How to contact you ?

### DISCUSSION THREAD

**Discussion by ghost** [aregtech/areg-sdk#467] [General]
you are my first follower in Github .thanks first .  I wonder how did you think about my Github?  why so less followers?  I am  a cpp coder in Automitove .,I work on. Automotive Vihicle IVI camera development. why did you follow my Github ? Later,how can I contact you  and talk about problems in C++ and C  and Python and so on.

**Comment by aregtech** [comment]
Hi @chenqiang2025 

nice to meet you :)
Actually, you are already contacted me. 
I would appreciate very much if you join the project and help.

**Reply by aregtech** [reply]
I moved this ticket to "Discussions".
